apps/src/code-studio/components/progress/ScriptOverviewTopRow.jsx

```diff
@@ -98,11 +117,24 @@ class ScriptOverviewTopRow extends React.Component {
             {assignedSectionId && <Assigned />}
           </div>
         )}
-        {!professionalLearningCourse &&
-          viewAs === ViewType.Teacher &&
-          resources.length > 0 && (
-            <TeacherResourcesDropdown resources={resources} unitId={scriptId} />
-          )}
+        {(!professionalLearningCourse || showCalendar) && (
```

Comment:
What happens if you leave out this outer condition?

Reply:
It'll just render an extra empty div

Reply:
actually, I just checked and it doesnt look too bad, I'll take it out